src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -307,7 +307,7 @@ namespace cryptonote
     uint64_t diff_v2 = max_block_height > last_block_v1 ? std::min(abs_diff, max_block_height - last_block_v1) : 0;
     assert(diff_v2 == 0);
     MCLOG(is_inital ? el::Level::Info : el::Level::Debug, "global", context <<  "Sync data returned a new top block candidate: " << m_core.get_current_blockchain_height() << " -> " << hshd.current_height
-      << " [Your node is " << abs_diff << " blocks (" << (diff_v2 / (24 * 60 * 60 / DIFFICULTY_TARGET_V2)) << " days) "
+      << " [Your node is " << abs_diff << " blocks (" << (diff / (24 * 60 * 60 / DIFFICULTY_TARGET_V2)) << " days) "
```

Comment:
The `diff` looks like it should be changed to `abs_diff` since the part following after is 
`current_height << Your node is X blocks X days << (0 <= diff) ? string("behind") : string("ahead")`

So on a negative difference, your blockchain is ahead of the target and the text will update accordingly to say that.